qovery/resource_helm.go

```diff
@@ -454,6 +454,34 @@ func (r helmResource) Schema(_ context.Context, _ resource.SchemaRequest, resp *
 					},
 				},
 			},
+			"custom_domains": schema.SetNestedAttribute{
+				Description: "List of custom domains linked to this helm.",
+				Optional:    true,
+				NestedObject: schema.NestedAttributeObject{
+					Attributes: map[string]schema.Attribute{
+						"id": schema.StringAttribute{
+							Description: "Id of the custom domain.",
+							Computed:    true,
+						},
+						"domain": schema.StringAttribute{
+							Description: "Your custom domain.",
+							Required:    true,
+						},
+						"generate_certificate": schema.BoolAttribute{
```

Comment:
_suggestion_: `Qovery will generate and manage the certificate for this domain.` => `Qovery will generate and manage domain's certificate.`